source/kraken/worker.cpp

```diff
@@ -751,14 +751,14 @@ void Worker::journeys(const pbnavitia::JourneysRequest& request, pbnavitia::API
                                            "no origin point nor destination point given");
             return;
         }
-
+        uint32_t max_extra_second_pass = 0;  // request.max_extra_second_pass()
```

Comment:
We don't use any more the parameter  max_extra_second_pass ?

Reply:
It is to deploy the raptor PR on all plaforms (dev, internal, etc.) without having to update the navitia-deployment-conf.
With the PR, the parameter should be zero, without the PR we want to keep using the parameters in navitia-deployment-conf.